File: Algo/hr_implement/2/letter_area_in_pdf.py
"""
Get area of letter for selecting in a PDF for given heights.
"""

def designerPdfViewer(h, word):
	#subtract value of alphabet from ascii is 97
	width = len(word)
	word_height = []
	for letter in word:
		word_height.append(h[ord(letter) - 97])
	return max(word_height) * width


if __name__ == "__main__":
	h = map(int,'1 3 1 3 1 4 1 3 2 5 5 5 5 5 5 5 5 5 5 5 5 5 5 5 5 7'.split(' '))
	word = 'zaba'
	print designerPdfViewer(h, word)

File: Algo/hr_implement/2/letter_area_in_pdf.js
function designerPdfViewer(h,word) {
    let width = word.length,
        word_height =[];
    
    word.split('').forEach(element => {        
        word_height.push(h[element.charCodeAt()-97]);
    });
    return (Math.max(...word_height)) * width;
}

let h = '1 3 1 3 1 4 1 3 2 5 5 5 5 5 5 5 5 5 5 5 5 5 5 5 5 7'.split(' ').map(Number),
	word = 'zaba';

console.log(designerPdfViewer(h, word));